content/guides/typescript.md

```diff
@@ -48,9 +60,13 @@ See [TypeScript's documentation](https://www.typescriptlang.org/docs/handbook/ts
 
 __webpack.config.js__
 
+To learn more about webpack configuration, see the [configuration concepts](/concepts/configuration/).
+
 Now let's configure webpack to handle TypeScript:
 
 ```js
```

Comment:
it's preferred to use single quotes everywhere for consistency. (the `extensions` array)